go/carmen/carmen.go

```diff
@@ -188,6 +189,16 @@ type HeadBlockContext interface {
 type HistoricBlockContext interface {
 	blockContext
 
+	// CreateWitnessProof creates a witness proof for the given account and keys.
+	// Error may be produced when it occurs in the underlying database,
+	// otherwise, the proof is returned.
+	CreateWitnessProof(address Address, keys ...Key) (WitnessProof, error)
```

Comment:
How about renaming this to just `GetProof`?

Reply:
done